lib/taste_tester/client.rb

```diff
@@ -125,14 +125,62 @@ def upload
 
     private
 
+    def tar_content(list, name)
+      full_paths = list.map do |path|
+        "#{TasteTester::Config.repo}/#{path}"
+      end
+      out = full_paths.map do |path|
+        if path.start_with?('/')
+          path = path[1..-1]
+        end
+        "--transform 's,#{path},#{name},'"
+      end
+      out.concat(
+        full_paths.each do |path|
+          "'#{path}'"
+        end,
+      )
+      # partial tar command line + list of full path used for finding
+      # symlinks
+      [out.join(' '), full_paths]
+    end
+
     def full
+      if TasteTester::Config.bundle
+        dest = "#{@server.bundle_dir}/tt.tgz"
```

Comment:
I do like the idea of keeping it always having one name, so we keep the same semantics of "an upload changes the state to the current repo"

Reply:
No action needed here